### PR TITLE
Add which-key git menu descriptions.

### PR DESCRIPTION
**Pull request by jmrah**
#17

## Description
Please explain the changes you made here.

## Checklist
- [x] I have read
[CONTRIBUTING](https://github.com/MarcoIeni/intellimacs/blob/master/docs/CONTRIBUTING.org)
guidelines.

#### [CHANGELOG](https://github.com/MarcoIeni/intellimacs/blob/master/docs/CHANGELOG.org):
- [ ] Updated
- [ ] I will update it after this PR has been discussed
- [x] No need to update

#### [KEYBINDINGS](https://github.com/MarcoIeni/intellimacs/blob/master/docs/KEYBINDINGS.org):
- [ ] Updated
- [ ] I will update it after this PR has been discussed
- [x] No need to update

#### [PLUGINS](https://github.com/MarcoIeni/intellimacs/blob/master/docs/PLUGINS.org):
- [ ] Updated
- [ ] I will update it after this PR has been discussed
- [x] No need to update

### CODE REVIEW

File: spacemacs/git-vcs.vim [hunk 1]
```diff
@@ -1,27 +1,42 @@
...
 nnoremap <leader>gi    :action Git.Init<CR>
 vnoremap <leader>gi    :action Git.Init<CR>
 
 " Show local changes (git status)
+let g:WhichKeyDesc_GitVcs_Status = "<leader>gs status"
 nnoremap <leader>gs    :action Vcs.Show.Local.Changes<CR>
 vnoremap <leader>gs    :action Vcs.Show.Local.Changes<CR>
 
+""" VCS Submenu ---------------------------------
+let g:WhichKeyDesc_GitVcs_Vcs = "<leader>gv +vcs"
```
Comment (jmrah) [inline spacemacs/git-vcs.vim]: I couldn't find the `gv` menu in my spacemacs.  I took a random guess at what the letter v stood for as `vcs`.  What does `v` actually stand for?

File: extra/git-vcs.vim [hunk 2]
```diff
@@ -1,19 +1,24 @@
 " Git checkout
+let g:WhichKeyDesc_GitVcs_Checkout = "<leader>gb checkout"
```
Comment (marcoieni) [inline extra/git-vcs.vim]:
```suggestion
let g:WhichKeyDesc_GitVcs_Branches = "<leader>gb branches"
```

I know the action is a checkout, but `b` stands for "branches" 😅
What do you think?
Maybe we call it `branch-checkout`?

File: spacemacs/git-vcs.vim (hunk 1 above)
Comment (marcoieni) [reply spacemacs/git-vcs.vim]: You have to add this layer in order to see it: https://develop.spacemacs.org/layers/+source-control/version-control/README.html

Comment (marcoieni) [inline spacemacs/git-vcs.vim]:
```suggestion
let g:WhichKeyDesc_GitVcs_VersionControl = "<leader>gv +version-control"
```

Comment (jmrah) [reply spacemacs/git-vcs.vim]: Thanks.  I added that layer, and see the 'v' menu now (stand for version-control)  Will update the PR.

File: extra/git-vcs.vim (hunk 2 above)
Comment (jmrah) [reply extra/git-vcs.vim]: Good catch.  TBH, not sure why I named it Checkout......let's stick with your suggested change.  I'll commit it.

File: spacemacs/git-vcs.vim (hunk 1 above)
Comment (jmrah) [reply spacemacs/git-vcs.vim]: I'm still getting used to Github PR's.  I didn't realize you could make an inline suggested code-change, and then I could commit that.  Very neat!

**Comment by marcoieni** [comment]
thanks!

**Pull request merged**